src/commit.rs

```diff
@@ -227,7 +230,7 @@ macro_rules! unicase_components {
     ($($ty:ident),+) => (
         $(
             /// A component of the conventional commit.
-            #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
+            #[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
```

Comment:
Why add `Default`?

Reply:
In the case of `#[serde(skip_deserializing)]` for `Commit`(since it doesn't implement `Deserialize`), `Default` implementation is needed.

Reply:
Without a `Deserialize` and any uses of skip_deserializing, I'm not seeing why this is needed. 

Also, I assume this defaults to an empty string. That isn't a valid default for some of the newtypes

Reply:
Yes it defaults to an empty string. It was basically for implementing `Default` on the `Commit`. It might have very few use cases without serde. (Frankly it looks like it's not needed at all but my code breaks without it :/)

How do we proceed here? What might be the valid `Default` implementation for `Commit` and its fields? Do you think having `Commit::default()` is a problem?

Reply:
Oops, I completely missed that you added `Default` to `Commit`.  The `PartialEq` makes sense.  We could probably make it `Eq`.

One option is to do `Option<Commit>`.  That'd better express what is going on.

Reply:
In case things are unclear, I'd like to see `Default` removed, at least from this PR.

Reply:
2fe294f